nikola/data/themes/base/templates/embedded-fotorama.tmpl

```diff
@@ -0,0 +1,16 @@
+## -*- coding: utf-8 -*-
+
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script> <!-- 33 KB -->
+
+<link  href="http://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.css" rel="stylesheet"> <!-- 3 KB -->
```

Comment:
You could also use the template hooks for that.

-- 
Chris Warrick <https://chriswarrick.com>
Sent from my Galaxy S3.
On 8 Jun 2015 12:23, "Roberto Alsina" <notifications@github.com> wrote:

> In nikola/data/themes/base/templates/embedded-fotorama.tmpl
> <https://github.com/getnikola/nikola/pull/1794#discussion_r31901731>:
>
> > @@ -0,0 +1,16 @@
> > +## -*- coding: utf-8 -*-
> > +
> > +<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script> <!-- 33 KB -->
> > +
> > +<link  href="http://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.css" rel="stylesheet"> <!-- 3 KB -->
>
> You can't drop CSS link elements in the middle of the HTML, it's not
> standards compliant (see Notes here:
> https://developer.mozilla.org/en/docs/Web/HTML/Element/link ).
>
> I'm afraid that means to support this you need to add that link element in
> your conf.py (EXTRA_HEAD is the option, IIRC)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getnikola/nikola/pull/1794/files#r31901731>.
>